core_api/src/database/seed.jest.ts

```diff
@@ -415,7 +415,19 @@ const seed = async () => {
         consultation2.consultationDate = '2024-11-22';
         const consultation3 = { ...baseConsultation };
         consultation3.consultationDate = '2024-11-13';
-        return [consultation1, consultation2, consultation3];
+        // Create a consultation for today, in the next 2 hours
```

Comment:
` // Create a consultation for today, in the next 2 hours`

ne faut-il pas rajouté aussi 1 rendez-vous 5min avant l'heure actuelle pour bien vérifier qu'elle ne s'affiche pas et 1 rendez-vous 2h30 après l'heure actuelle pour aussi s'assuré qu'on ne l'affiche pas ?